### DISCUSSION TITLE
Welcome to higress Discussions!

### DISCUSSION THREAD

**Discussion by johnlanni** [higress-group/higress#102] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cuiyongchao** [comment]
higress对接consul注册中心

**Comment by liyongxian** [comment]
higress 性能对标 nginx ingress ，在qps和延迟方面，有测试数据吗？多谢。
